test/emailpassword/signupFeature.test.js

```diff
@@ -1093,10 +1093,6 @@ describe(`signupFeature: ${printPath("[test/emailpassword/signupFeature.test.js]
                             id: "email",
                             value: "random@gmail.com",
                         },
-                        {
-                            id: "testField",
-                            value: "",
-                        },
```

Comment:
why is this removed?

Reply:
To fix the test. This test is to make sure that we get an error when the required fields are not passed.

The fix was either to remove this (which would throw an error) or add 2 new ones (which would also throw an error)